React/Modules/RCTUIManager.m

```diff
@@ -399,6 +399,21 @@ - (void)setFrame:(CGRect)frame forView:(UIView *)view
   });
 }
 
+- (void)setIntrinsicContentSize:(CGSize)size forView:(UIView *)view
+{
+  RCTAssertMainThread();
+
+  NSNumber *reactTag = view.reactTag;
+  dispatch_async(_shadowQueue, ^{
+    RCTShadowView *rootShadowView = _shadowViewRegistry[reactTag];
```

Comment:
Hah, yeah, totally agreed, I made it reflect the code in `setFrame:forView:`, which clearly since then has been updated to also remove the ‘root’ part and I just didn’t see that after rebasing. Will update.